components/resolved_ts/src/endpoint.rs

```diff
@@ -806,19 +815,21 @@ where
                 }
             }
         }
+        let now = TimeStamp::physical_now();
```

Comment:
Can we use PD time to calculate the safe ts gap(it's fetched when `advance_ts_for_regions` is called, add the timeout duration to reduce error)? If the resolved ts interval is tenths or hundreths, the accuracy may be a problem.

Reply:
I'm hesitating about introducing an RPC call here. The result would be affected by RPC and could also be affected by the runtime that drives the RPC.. Seems less reliable to me for a metric service. What do you think?